client_python/client_python.py

```diff
@@ -0,0 +1,54 @@
+import time
+
+import requests
+import json
+from datetime import datetime
+from status import Status
+
+
+class FileUploadClient:
+    def __init__(self, base_url):
+        self.base_url = base_url
+
+    def upload(self, file_path):
+        url = self.base_url + '/upload'
+        with open(file_path, 'rb') as file:
+            response = requests.post(url, files={'file': file})
+            if response.status_code == 200:
+                response = json.loads(response.text)
+                uid = response['UID']
+                return uid
+            else:
+                raise Exception(f"File upload failed with status code: {response.status_code}")
```

Comment:
Always throw the most specific exception -
https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python
https://docs.python.org/3/library/exceptions.html